CMakeLists.txt

```diff
@@ -49,9 +49,6 @@ if(NOT CRASHREPORTER_EXECUTABLE)
     set(CRASHREPORTER_EXECUTABLE "${APPLICATION_EXECUTABLE}_crash_reporter")
 endif()
 
-set(synclib_NAME "${APPLICATION_EXECUTABLE}sync")
-set(csync_NAME "${APPLICATION_EXECUTABLE}_csync")
```

Comment:
That's in general a good idea. However, we do branding for our customers and it's important that the executable file names follow their branding options. I think it is possible to set the output name regardless of the target name with `target_set_properties`. See https://cmake.org/cmake/help/latest/prop_tgt/OUTPUT_NAME.html. Could you please investigate that and change it if possible?

Reply:
right, I did exactly that for synclib (https://github.com/nextcloud/desktop/pull/3110/files#diff-77dc0d12313e03cf60d8966b79d1c11005d2ae20cea3fa75464744e7484e7efdR116) but forgot about the executables